src/renderer/containers/Onboard/CreatePassword/CreatePasswordContainer.tsx

```diff
@@ -0,0 +1,101 @@
+import {MultipleInputVertical} from "../../../components/MultipleInputVertical/MultipleInputVertical";
+import {RouteComponentProps} from "react-router";
+import {passwordFormSchema} from "./validation";
+import {joiValidationToErrorDetailsMessages} from "../../../services/validation/util";
+import {joinArrayOxfStyle} from "../../../services/utils/utils";
+
+export interface IState {
+    password: string;
+    confirm: string;
+    errorMessages: {
+        password?: string,
+        confirm?: string
+    };
+}
+
+export class CreatePasswordContainer extends Component<Pick<RouteComponentProps, "history">> {
+
+    public state: IState = {
+        password: "",
+        confirm: "",
+        errorMessages: {}
+    };
+
+    public render(): ReactElement {
+        const inputs: Array<IInputFormProps> = [
+            {
+                inputId:"inputPassword",
+                onChange: this.handleChange,
+                placeholder: "Enter password",
+                valid: this.isValid(this.state.errorMessages.password),
+                errorMessage: this.state.errorMessages.password
+            },
+            {
+                inputId:"confirmPassword",
+                onChange: this.handleChange,
+                placeholder: "Confirm password",
+                valid: this.isValid(this.state.errorMessages.confirm),
+                errorMessage: this.state.errorMessages.confirm
+            }
+        ];
+        return (
+            <>
+                <h1>Create a password</h1>
+                <p>You will use this password to unlock applications and keys.</p>
+                <div className="input-container input-container-vertical">
+                    <MultipleInputVertical inputs={inputs}/>
+                    <ButtonPrimary
+                        buttonId="next"
+                        disabled={ this.state.errorMessages.password !== "" || this.state.errorMessages.confirm !== ""}
+                    >
+                        NEXT
+                    </ButtonPrimary>
+                </div>
+            </>
+        );
+    }
+
+    private isValid(error: string | undefined): boolean | undefined {
+        return (typeof error === "undefined") ? error : (error === "");
+    }
+
+    private handleChange = (e: React.FormEvent<HTMLInputElement>): void => {
+        const selector = (e.currentTarget.id === "inputPassword") ? "password" : "confirm";
+        const localState = this.state;
+        localState[selector] = e.currentTarget.value;
+        this.setState({[selector]: e.currentTarget.value});
+        // validate state
+        const validation = passwordFormSchema.validate(localState, {abortEarly: false});
+        // generate error messages
+        const m: {password?: string, confirm?: string} = {password: "", confirm: ""};
+        if (validation.error) {
+            const errors = joiValidationToErrorDetailsMessages(validation.error);
```

Comment:
Just display those errors, why do you have all code below?

We cannot merge this method as it's way too huge and unmaintainable. Just make it as simple as possible, we will worry about UX later on,